README.rst

```diff
@@ -315,6 +315,11 @@ Also you can specify ``sentry-dsn`` parameter when running rqworker:
 ``./manage.py rqworker --sentry-dsn=https://*****@sentry.io/222222``
 
 
+**Disable RQ sentry plugin**
+
+If your project use ``sentry-sdk``, the DSN is not compatible with RQ's sentry plugin (based on raven).
```

Comment:
RQ's Sentry plugin is not based on Raven. The rationale to disable this should be modified to reflect this comment https://github.com/rq/django-rq/issues/366#issuecomment-552860824 (i.e having custom sentry-sdk instantiation).